pyproject.toml

```diff
@@ -21,7 +21,7 @@ Changelog = "https://github.com/r4fek/django-scylla/releases"
 
 [tool.poetry.dependencies]
 python = "^3.8"
-django = "^3.1 || ^3.2 || ^4.0 || ^4.1"
+django = ">=3.1 <4.2"
```

Comment:
As noted in the related commit, Django 4.2 breaks this adapter due to an exception added in as a query hint that warns when a query would select all tables.

See: https://docs.djangoproject.com/en/4.2/_modules/django/core/exceptions/#FullResultSet